awesomemotive/all-in-one-seo-pack
# Add support for Image SEO module

Resolves #3146

## Proposed changes

This PR adds the code that has to go in free to support the Image SEO module.

As an extra, I've fixed the flags on the Feature Manager screen and added CTA buttons to upgrade to the premium version - 

![image](https://user-images.githubusercontent.com/18156697/75112905-f4a04680-5648-11ea-9e0a-b696d7acd6da.png)

## Types of changes

- New feature (non-breaking change which adds functionality)
- Improves existing functionality

## Checklist


## Testing instructions

This has to be tested after it's built as Pro (see PR in Pro repo) but also as the free version:

1. Install the PR and go to the Feature Manager screen. 
There should be no notices and the Video Sitemap module and Image SEO module should have two orange CTA buttons. The flags should both say "PRO".
2. Check the link for both of the upgrade buttons. The structure should be the same except for the "utm_content" param which should be different and state the name of the module.
3. Check that plugin doesn't throw any errors when the Image SEO module is activated in Pro, and when you then deactivate Pro and activate the free version. Same goes for the Video Sitemap module.
